plaso/cli/tools.py

```diff
@@ -507,3 +511,38 @@ def Write(self, string):
       # sys.stdout.write() on Python 3 by default will error if string is
       # of type bytes.
       sys.stdout.write(string)
+
+
+class FindSpecsGetter(object):
```

Comment:
several nits:
* please don't overload the term getter (also see: https://en.wikipedia.org/wiki/Mutator_method)
* I opt we make this a method part of BaseEngine instead?